NEWS.md

```diff
@@ -1,5 +1,7 @@
 # pacta.portfolio.audit (development version)
 
+* calculate the number of shares for component holdings of funds when they are expanded, which results in an appropriate ownership weight being calculated (rather than always 0 as before) for equity indirectly held through a fund (#12)
```

Comment:
Nit: capitalize the "c"?

Reply:
```suggestion
* Calculate the number of shares for component holdings of funds when they are expanded, which results in an appropriate ownership weight being calculated (rather than always 0 as before) for equity indirectly held through a fund (#12)
```